stock_deposit/models/product.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# © 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, fields, models
+from openerp.tools.safe_eval import safe_eval as eval
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    @api.multi
+    def _compute_deposit_available(self):
+        for product in self:
+            product.deposit_available = product.with_context(
+                deposit_locations=True).qty_available
+
+    deposit_available = fields.Float(
+        compute='_compute_deposit_available',
+        string='Deposit Qty Available'
+    )
+
+    def _exclude_deposit_location_action(self, res_action):
+        ctx = eval(res_action['context'])
+        ctx.update({'search_default_deposit_loc_exclude': 1})
+        res_action['context'] = ctx
+        return res_action
+
+    @api.multi
+    def action_view_stock_moves(self):
+        res = super(ProductTemplate, self).action_view_stock_moves()
+        return self._exclude_deposit_location_action(res)
+
+    @api.multi
+    def action_open_quants(self):
+        res = super(ProductTemplate, self).action_open_quants()
+        return self._exclude_deposit_location_action(res)
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    @api.multi
+    def _compute_deposit_available(self):
```

Comment:
The dictionary cannot be completed as there's no prepare method for this dictionary, but the implementation in https://github.com/OCA/stock-logistics-workflow/pull/268/commits/96d98054debbf09337d93c834fdcd222432b91fd is more efficient than previous one.